app/src/main/res/values/strings.xml

```diff
@@ -9,4 +9,10 @@
     <string name="facebook_app_id">1008042499278780</string>
 
     <string name="create_post">Create post</string>
+
+    <string name="post_text">Post Text</string>
```

Comment:
nit: Could you add a single comment saying the activity where these strings are used? It'll be useful in the future as we get a lot more strings!